.travis.yml

```diff
@@ -1,38 +1,40 @@
 language: node_js
 node_js:
-  - "4"
+  - "6"
 
 sudo: false
 
 cache:
   directories:
-    - node_modules
+    - $HOME/.npm
+    - $HOME/.cache # includes bowers cache
```

Comment:
do we need bower's cache (and similarily do we need to npm install bower below in the `before_install` block or run `bower install` in the `install block?)

Reply:
I left the cache there since Bower is still used by ember-try (I assume it's using the cache, but not totally sure), but removed the Bower install from `before_install` and the `bower install` from the `install` block so it's just handled by ember-try.